ports/winit/browser.rs

```diff
@@ -437,6 +456,7 @@ where
                 },
                 EmbedderMsg::HistoryChanged(urls, current) => {
                     self.current_url = Some(urls[current].clone());
+                    self.current_url_taken = false;
```

Comment:
Maybe just store the stringified version here.

Reply:
done in b01b10c591801be4736ccd501cfa292c9a361bc7